scripts/ci-checks.sh

```diff
@@ -12,6 +12,9 @@ fi
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
+ROOT_DIR=$( dirname "$SCRIPT_DIR" )
+pushd "$ROOT_DIR"
```

Comment:
Without a `popd`, would this leave the caller in `$ROOT_DIR` too?

Reply:
No - this dir stack is local to the script, so when we exit (exceptionally or otherwise) the stack is unchanged and the caller remains where they were.